app/views/DR/ReportScreen/sections/SetpAdress.js

```diff
@@ -143,6 +161,7 @@ const StepAdress = ({ setCompleted }) => {
                   } else if (text === '') {
                     setSelectedOption('numberPersonLivesWith', '');
                   }
+                  getCurrentLocation();
```

Comment:
Esto se envia para el formulara a las respuestas al final y se salva en el stater tambien por si se necesita en el otro screem ??

Reply:
Se envia al global state directamente, si se necesita en otro screen solo habría que acceder al global state(a su latitud o longitud)

Reply:
Nice this is done